platform/unix/process.rs

```diff
@@ -26,13 +27,11 @@ pub fn exec(command: &Command) -> io::Error {
 
     let env: Vec<_> =
         command.env.iter().map(|(key, value)| {
-            format!("{}={}",
-                    String::from_utf8(key.to_bytes().to_vec()).unwrap(),
-                    String::from_utf8(value.to_bytes().to_vec()).unwrap())
+            let entry = format!("{}={}",
+                                str::from_utf8(key.to_bytes()).unwrap(),
+                                str::from_utf8(value.to_bytes()).unwrap());
+            CString::new(entry).unwrap()
```

Comment:
Nevermind. I think I was originally thinking that `execve` would release the strings that were passed in